kani-driver/src/call_cargo.rs

```diff
@@ -87,8 +87,8 @@ impl KaniSession {
         // Only joing them at the end. All kani flags must come first.
         kani_args.extend_from_slice(&rustc_args);
 
+        let mut any_target = false;
```

Comment:
How about renaming this variable into a more readable one like `target_found` or `supported_target_found`?

Reply:
Yeah, that makes sense.